src/NServiceBus.Gateway.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -89,9 +88,10 @@ namespace NServiceBus
 {
     public static class GatewayExtensions
     {
-        public static NServiceBus.GatewaySettings Gateway(this NServiceBus.EndpointConfiguration config, NServiceBus.Gateway.GatewayDeduplicationConfiguration storageConfiguration) { }
+        public static NServiceBus.GatewaySettings Gateway<TGatewayDeduplicationConfiguration>(this NServiceBus.EndpointConfiguration config, TGatewayDeduplicationConfiguration storageConfiguration)
```

Comment:
I assume here that this API change will be invisible to users as the same code will satisfy the generic variant.